src/python/pcmdi/__init__.py

```diff
@@ -1,4 +1,6 @@
 import io
 from mean_climate_metrics_calculations import compute_metrics
+from mean_climate_metrics_calculations_ExtInfo import compute_metrics_ExtInfo
```

Comment:
@durack1 @gleckler1 do we want these calculation as part of our distribution? Or should this only be installed at IPSL where they need it?